backend/controllers/articleController.js

```diff
@@ -41,12 +41,22 @@ const getById = async (req, res) => {
 
 const create = async (req, res) => {
   const { title, body, summary } = req.body;
+  console.log(body.length, process.env.MAX_BODY)
+  if (title.length > process.env.MAX_TITLE) {
+    return res.status(400).json({ error: `Your title should be less than ${process.env.MAX_TITLE} characters.` });
+  }
+  if (summary && summary.length > process.env.MAX_SUMMARY) {
```

Comment:
Why here testing if `summary` exists before checking length but not for `body`?

Reply:
Title and body are `required` in the article model but summary is not

Reply:
Ok tks!